src/imitation/algorithms/adversarial/common.py

```diff
@@ -212,6 +225,7 @@ def __init__(
             self.venv_wrapped = venv
             self.gen_callback = None
         else:
+            # TODO(juan) this is a bug; venv is going unused.
             venv = self.venv_wrapped = reward_wrapper.RewardVecEnvWrapper(
```

Comment:
Yeah think we can just delete `venv` here and a few lines above (though note `self.venv_wrapped` is being used, so it's mostly a cosmetic issue).